Codeinwp/neve
# Fix current_post_meta magic tag

### Summary
The content was escaped from span and time tags. I changed the escape function to accept those if it's current_post_meta magic tag.

### Will affect the visual aspect of the product
NO


### Test instructions
- Create new custom layout and add {current_post_meta} in its content.
- Make the layout to appear only if the post type is post
- Make a new post, publish it, then edit it again and save it
- Check it on the frontend. The edited time should be in the markup but should not be visible


Closes Codeinwp/neve-pro-addon#878.
